continuous_integration/travis/install.sh

```diff
@@ -19,6 +19,7 @@ if [ "$USE_CONDA" = "yes" ]; then
 
     # Install main dependencies
     conda install -q -y -c spyder-ide --file requirements/conda.txt
+    conda install -q -y -c spyder-ide --file requirements/conda_linux.txt
```

Comment:
Let's remove this and simply add it to `conda.txt`. Given that pyxdg is a noarch package, we can install it on Windows and we'll avoid making our Contributing guide more complex for no real gain.